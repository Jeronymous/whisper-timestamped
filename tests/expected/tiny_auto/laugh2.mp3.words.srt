1
00:00:00,140 --> 00:00:00,400
Thank

2
00:00:00,400 --> 00:00:00,700
you.

