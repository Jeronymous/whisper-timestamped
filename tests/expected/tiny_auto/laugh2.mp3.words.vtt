WEBVTT

00:00.140 --> 00:00.400
Thank

00:00.400 --> 00:00.700
you.

