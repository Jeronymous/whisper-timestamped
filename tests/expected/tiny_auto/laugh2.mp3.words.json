{
  "text": " Haha.",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.18,
      "end": 0.56,
      "text": " Haha.",
      "tokens": [
        50364,
        19131,
        13,
        50401
      ],
      "temperature": 0.0,
      "avg_logprob": -1.316188430786133,
      "compression_ratio": 0.38461538461538464,
      "no_speech_prob": 0.5721203088760376,
      "confidence": 0.079,
      "words": [
        {
          "text": "Haha.",
          "start": 0.18,
          "end": 0.56,
          "confidence": 0.079
        }
      ]
    }
  ],
  "language": "en",
  "language_probs": {
    "en": 0.6548641920089722,
    "zh": 0.038717932999134064,
    "de": 0.025590991601347923,
    "es": 0.011994256637990475,
    "ru": 0.011355933733284473,
    "ko": 0.04633929580450058,
    "fr": 0.02422906644642353,
    "ja": 0.014243531040847301,
    "pt": 0.019468599930405617,
    "tr": 0.0092684431001544,
    "pl": 0.007989896461367607,
    "ca": 0.0003766233567148447,
    "nl": 0.02767045982182026,
    "ar": 0.0034096697345376015,
    "sv": 0.004660465754568577,
    "it": 0.0023070976603776217,
    "id": 0.0032282101456075907,
    "hi": 0.019166767597198486,
    "fi": 0.00038405158556997776,
    "vi": 0.0028937491588294506,
    "he": 0.0006482097669504583,
    "uk": 0.0007608000887557864,
    "el": 0.0010562663665041327,
    "ms": 0.0037156471516937017,
    "cs": 0.0018179493490606546,
    "ro": 0.003383135423064232,
    "da": 0.001361581147648394,
    "hu": 0.001269133179448545,
    "ta": 0.0015980799216777086,
    "no": 0.0012741003884002566,
    "th": 0.00850520096719265,
    "ur": 0.002117111813277006,
    "hr": 0.0002384630497545004,
    "bg": 0.00032088710577227175,
    "lt": 0.00015098540461622179,
    "la": 0.001875657238997519,
    "mi": 0.0015489121433347464,
    "ml": 0.001011841231957078,
    "cy": 0.01327643170952797,
    "sk": 0.0001626176672289148,
    "te": 0.0011112901847809553,
    "fa": 0.0003551896952558309,
    "lv": 0.00010540479706833139,
    "bn": 0.0016813284019008279,
    "sr": 9.122024493990466e-05,
    "az": 9.979525930248201e-05,
    "sl": 0.0002431663015158847,
    "kn": 2.787499215628486e-05,
    "et": 4.0085364162223414e-05,
    "mk": 3.091512917308137e-05,
    "br": 0.0008587384945712984,
    "eu": 0.00010057795589091256,
    "is": 0.00015577819431200624,
    "hy": 0.00014748783723916858,
    "ne": 0.00018141313921660185,
    "mn": 0.0002681112091522664,
    "bs": 0.0002431663015158847,
    "kk": 2.7335841878084466e-05,
    "sq": 6.172270514070988e-05,
    "sw": 0.00029909968725405633,
    "gl": 0.0007259605336003006,
    "mr": 0.00017310561088379472,
    "pa": 8.081665873760357e-05,
    "si": 0.0005147830815985799,
    "km": 0.0007637778180651367,
    "sn": 0.000816219428088516,
    "yo": 0.00030619261087849736,
    "so": 3.4114727895939723e-06,
    "af": 4.6590746933361515e-05,
    "oc": 0.00016325415344908834,
    "ka": 4.385352076496929e-05,
    "be": 7.833019481040537e-05,
    "tg": 2.4760939254520054e-07,
    "sd": 0.00022054229339119047,
    "gu": 8.875974162947387e-05,
    "am": 1.856874951045029e-05,
    "yi": 0.00012468329805415124,
    "lo": 5.362561932997778e-05,
    "uz": 3.726850383856117e-09,
    "fo": 0.00023022523964755237,
    "ht": 0.0003962427144870162,
    "ps": 4.604794958140701e-05,
    "tk": 2.9428679937382185e-08,
    "nn": 0.00637010158970952,
    "mt": 3.388737604836933e-05,
    "sa": 0.00042015372309833765,
    "lb": 1.55556463710127e-07,
    "my": 0.00015885064203757793,
    "bo": 0.0005587917985394597,
    "tl": 0.0009579920442774892,
    "mg": 3.7639857453086734e-08,
    "as": 4.351224924903363e-05,
    "tt": 1.580061166350788e-07,
    "haw": 0.0029623722657561302,
    "ln": 1.3479466360877268e-05,
    "ha": 5.124700308556385e-08,
    "ba": 2.5718332352653306e-08,
    "jw": 0.0032030881848186255,
    "su": 1.0352179913297732e-07
  }
}